### ISSUE TITLE
Console output is unreadable when including CJK characters of shell run

### ISSUE THREAD

**Issue by xloon** [tekartik/process_run.dart#31]
Example code
```dart
var shell  = Shell();
await shell.run('echo 中文/한국어/にほんご');
```

Output
```
$ echo -�/\m�/k{�T
中文/한국어/にほんご
```

**Comment by alextekartik** [comment]
Thanks for the report. Indeed the encoding was missing. I published 0.11.2+6 that should fix the issue

**Comment by xloon** [comment]
Sorry for the late response, it's working all right now.
